docs/control-structures.rst

```diff
@@ -272,7 +272,10 @@ Assignment
 Destructuring Assignments and Returning Multiple Values
 -------------------------------------------------------
 
-Solidity internally allows tuple types, i.e. a list of objects of potentially different types whose size is a constant at compile-time. Those tuples can be used to return multiple values at the same time and also assign them to multiple variables (or LValues in general) at the same time::
+Solidity internally allows tuple types, i.e. a list of objects of potentially different types whose size is a constant at compile-time. Those tuples can be used to return multiple values at the same time.
+These can then either be assigned to newly declared variables or to pre-existing variables (or LValues in general):
+
+::
 
     pragma solidity ^0.4.16;
```

Comment:
Needs a version bump.

Reply:
This was missed.

Reply:
@chriseth this version bump is still missing :)

Reply:
argh, sorry.